server/cmwell-ws/app/controllers/Application.scala

```diff
@@ -855,7 +855,7 @@ callback=< [URL] >
                             if (bytes.isEmpty) ""
                             else all.lines.next()
                           }
-                          logger.info(s"""[$id] onPush(first line: "$elem", num of lines: ${all.lines.size}, num of chars: ${all.length})""")
+                          logger.debug(s"""[$id] onPush(first line: "$elem", num of lines: ${all.lines.size}, num of chars: ${all.length})""")
```

Comment:
only if `debugLog==true`, and we want it info level

Reply:
didn't realize it was only when debugLog==true. will fix it.